modules/subscriptions.php

```diff
@@ -567,6 +567,26 @@ function Jetpack_Subscriptions_Widget() {
 		$this->WP_Widget( 'blog_subscription', __( 'Blog Subscriptions (Jetpack)', 'jetpack' ), $widget_ops, $control_ops );
 	}
 
+	/*
+	 * Check for the option to show subscriber count.
+	 *
+	 * This function only serves the purpose of backwards compatibility for versions < 3.4
+	 *
+	 * @since 3.5
+	 * @return bool  Will return true only if a site still has the option to show subscriber count.
+	 */
+	function has_show_subscriber_count_option() {
+		$widget_options = get_option( 'widget_blog_subscription' );
```

Comment:
I was thinking we should keep the option (and the subscriber count) there until the user saves the widget again.  This would be done by <a href="https://github.com/Automattic/jetpack/pull/1967/files#diff-6e4337c8315c90dd70aea03475af1dc1R757">L757</a>